charts/bookkeeper/values.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: "bookkeeper.pravega.io/v1alpha1"
+kind: "BookkeeperCluster"
+metadata:
+  name: "pravega"
```

Comment:
This is not a pravega cluster. Please use a suitable name ... perhaps "pravega-bk"

Reply:
done